gems/aws-sdk-core/spec/aws/credential_resolution_chain_spec.rb

```diff
@@ -84,7 +84,7 @@ module Aws
           'AR_TOKEN'
         )
         client = ApiHelper.sample_rest_xml::Client.new(
-          profile: 'ar_web_identity', region: 'us-west-2'
+          profile: 'ar_web_identity', region: 'us-east-1'
```

Comment:
Why `us-east-1` here? Even using us-west-2 and setting regional we should expect us-west-2 in the url.

Reply:
The point was to have a consistent region across all of the tests to avoid having to mock responses from multiple endpoints - about half of them were us-east-1 and half were us-west-2 so I arbitrarily picked us-east-1, but I can easily change that if you have a preference.

Reply:
No preference! I see what you're doing now.